person.rb

```diff
@@ -0,0 +1,22 @@
+# person.rb
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknown')
+    @id = id
```

Comment:
- [ ] [REQUIRED] Good job so far! The `line 7` `:id` should only be able to be retrieved so only the `Getter` is defined 🙏🏼 Please implement like this:

```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
```